.gitignore

```diff
@@ -237,3 +238,8 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+
+# custom
+/output/**
+/test/test_output/**
```

Comment:
I needed to add this directory myself, so not sure if this should be in the gitignore

Reply:
This should be in the gitignore. However as Lukas supposed, the tests should be able to create it on demand.